### ISSUE TITLE
Multiple Instances of the applicaiton opened on the same platform is not updated

### ISSUE THREAD

**Issue by nus-se-script** [nus-cs2103-AY2425S1/pe-dev-response#628]
**ISSUE**

In EventfulNUS, when multiple instances of the application are open, changes made in one window do not reflect in the other. This can lead to data inconsistency, particularly for users who manage a high volume of event data across multiple faculties, roles, or participants[which is mentioned in the UG and DG]. A user might open multiple instances of EventfulNUS, intending to work on different aspects of the event data in parallel (e.g., managing participants in one window while scheduling events in another). However, any changes made in one instance, such as adding a new participant or updating event details, remain isolated to that specific window. 
The manager for this should be very busy also, most people working/ study have multiple tabs like 100 tabs open, not to mention about people working to handle the IFG, a cross-faculty University level game
This could lead the user to mistakenly assume that their updates were applied globally across all instances.

Further, there was mentioning in the UG that the data is saved automatically, which could be confusing to users.

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/b68c2505-51bc-4342-a3af-962219dc365b.png)

**Demostration**
<i><video controls><source src="https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/ae9079c3-7d51-4fac-a962-7bd253a2e010.mp4" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/ae9079c3-7d51-4fac-a962-7bd253a2e010.mp4</i>

**Reason for severity**

This issue impacts the reliability of the application, particularly in event management scenarios where accuracy and synchronization are essential, such as coordinating participants, updating event details, or assigning roles. For busy event organisers, managing a large volume of data across faculties or roles, the risk of data discrepancies becomes particularly significant.

<hr>
<sub>[original: nus-cs2103-AY2425S1/pe-interim#888] [original labels: type.FeatureFlaw severity.High]</sub>

**Comment by leowolf275** [comment]
# Team's Response
Hello, thank you for the bug report! While we know that users may want to use multiple instances of the app and edit the lists in parallel, we see this feature as **"Out Of Scope"** for the current version of our project. For now, we believe it's suitable for our users to stick to one application instance at a time.

In addition, this would violate the Constraint-Single-File. According to our project constraints, everything should be packaged into one single JAR file. 

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe-dev-response/main/files/92d9cc23-842e-4679-ab9d-b08a1b099273.png)

This is definitely a feature that we can consider implementing for future iterations of the project!




## Duplicate status (if any):
--
